### ISSUE TITLE
[New Sensor]: Jaalee Thermometer

### ISSUE THREAD

**Issue by wanfr** [custom-components/ble_monitor#976]
### Sensor Description

jaalee JHT Temperature/Humidity/Dewpoint/VPD Sensor

### Additional information

https://www.aliexpress.com/item/10000335836820.html
https://jaalee.fr.aliexpress.com/store/5869524

### BLE advertisements

```shell
Hello
I'm not a computer scientist and I can't give you more information.
With the ios application (jaalee smart) it gives me information temperature (°c), humidity (%), pressure (kPa) and devpoint (°C) just
With HomeAssistant the values are all wrong
Example: app -> 4.22°C HA -> 97.3°C

thank you for your understanding and help

CUSTOM SERVICE:
0xFFB0, 0x182F, 0x1802, 0x180A, 0xFF80, 0xFF70, 0xFF60, 0xFF50, 0xFF40, 0xFF30, 0xFFF0
TX POWER:
0x1804
```

**Comment by Ernst79** [comment]
What do you mean with HA giving the wrong data? As what device (brand) is it recognized?

**Comment by wanfr** [comment]
he is recognized for APPLE iBeacon Firmware: iBeacon

**Comment by Ernst79** [comment]
Than it is probably using their own format (Jaalee's own format). If you want me to figure out if it is possible to decode the data, I will need some data. To get this, follow these instructions. 

https://custom-components.github.io/ble_monitor/sensor_request

**Comment by SBastrykin** [comment]
Good afternoon. The same sensor. Correctly shows humidity, temperature is a random number.
Registered in configuration.yml logger. Tell me where to view the information

**Comment by Ernst79** [comment]
If you have enabled debug logging for BLE monitor and enabled the option "report_unknown", just make a copy of your Home Assistant Logbook (Settings - Sytem - Logbook)

**Comment by MrVaKo** [comment]
[Bluetooth LE Device GATT Results (jaalee, E3 6A 3D 3D C3 B4).txt](https://github.com/custom-components/ble_monitor/files/9523273/Bluetooth.LE.Device.GATT.Results.jaalee.E3.6A.3D.3D.C3.B4.txt)

This is taken from a Jaalee JHT-2 [±0.1C ±1.5%] sensor. The app shows temp, humidity, vpd and dew point. 

If you need something different, let me know as I'm very keen to get this working in HA.

**Comment by 2x4ever** [comment]
> If you have enabled debug logging for BLE monitor and enabled the option "report_unknown", just make a copy of your Home Assistant Logbook (Settings - Sytem - Logbook)

Hello, Ernst79.
MrVaKo provided gatt results. Is it enough decode Jaalee proto or do you need more info?
I am also very interested in this sensor.
Thanks

**Comment by Ernst79** [comment]
I actually need a normal log, not the GATT characteristics. GATT characteristiscs are for devices that need a connection. while we need to know if the device is sending passive BLE data. You can check this with the instructions here. 

https://custom-components.github.io/ble_monitor/sensor_request

**Comment by Ernst79** [comment]
I think this issue can be closed, added in 11.0.0

**Comment by dan-s-github** [comment]
@2x4ever do you by coincidence have a HA bluetooth proxy running and could extract some HA logs from the device that would include the manufacturer data? This might help identifying the Jaalee sensors types, just guessing here but @Ernst79 is investigating in regards to my sensors (see mentioned issue above).
